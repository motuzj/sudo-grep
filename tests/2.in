one word search + ignore case
"risus"
