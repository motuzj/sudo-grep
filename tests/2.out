Sit amet risus nullam eget felis eget nunc lobortis.
